test/features/perform_sanity_checks_on_reviews_test.rb

```diff
@@ -0,0 +1,194 @@
+require 'test_helper'
+Dir[Rails.root.join('test/**/*.rb')].each { |f| require f }
+
+class PerformsSanityChecksOnReviewsTest < Capybara::Rails::TestCase
+  include Helpers::UserHelpers
+
+  before do
+    stub_request(:get, "#{Cmr.get_cmr_base_url}/search/granules.echo10?concept_id=G309210-GHRC")
+      .with(
+        headers: {'Accept-Encoding' => 'gzip;q=1.0,deflate;q=0.6,identity;q=0.3', 'Accept' => '*/*', 'User-Agent' => 'Ruby'}
+      )
+      .to_return(status: 200, body: '<?xml version="1.0" encoding="UTF-8"?><results><hits>0</hits><took>32</took></results>', headers: {})
+  end
+
+  describe 'performs "mark complete" on a collection record' do
```

Comment:
I really like the idea of more feature tests for dashboard.  It's a huge help to people coming onto a new app to have some baseline of 'did I break it?'